### ISSUE TITLE
Can the name be split into two fields?

### ISSUE THREAD

**Issue by judgej** [kenkeiter/skeuocard#153]
I would like to use this with Authorize.Net DPM. The DPM accepts a first name and last name as two separate fields, and it does not support a single cardholder field. Is there an option built into skeuocard to allow for two name fields? Or would this require some major customisation?

Additionally, Authorize.Net DPM takes the expiry date as a single, four-digit field. Combining the two fields that skeuocard uses is trivial. It is the opposite direction with the free-format name field that is probably not going to be easy.

Any ready solutions to this?

**Comment by judgej** [comment]
We could split the name that is entered into first and surname, but I imagine that would start to get tricky with double-barrel names, and when titles are added ("MR FREDERICK LORD BLENKINSOP").

**Comment by micfacch** [comment]
@judgej did you find or create a solution to this issue?

**Comment by judgej** [comment]
No, we used the JS library from Stripe instead. That works well enough.
